src/resources/index.ts

```diff
@@ -1,21 +0,0 @@
-import { BenefitCompanyStatementResource } from './Correction'
-import { BenefitCompanyResource, CooperativeResource } from './Alteration'
-import { SoleProprietorshipResource, GeneralPartnershipResource } from './Change'
-import { ResourceIF } from '@/interfaces/'
-
-export * from './Correction'
-export * from './BreadCrumbResource'
-
-export const AlterationResources: Array<ResourceIF> = [
-  BenefitCompanyResource,
-  CooperativeResource
-]
-
-export const ChangeFirmResources: Array<ResourceIF> = [
-  SoleProprietorshipResource,
-  GeneralPartnershipResource
-]
-
-export const CorrectionResources: Array<any> = [
-  BenefitCompanyStatementResource
-]
```

Comment:
Yes, I deleted this file. Firstly, I was getting resource name conflicts. Secondly, I think it's more clear for the various views to just import from the sub-folder they care about (Alteration, Change, Conversion, Correction).